src/Schema/Events/Click.ts

```diff
@@ -513,30 +513,53 @@ export interface ClickedOnArtworkShippingWeight {
  * ```
  */
 export interface ClickedOnArtworkShippingUnitsDropdown {
-  action: ActionType.clickedNavigationTab
+  action: ActionType.clickedOnArtworkShippingUnitsDropdown
   context_module: ContextModule
   context_page_owner_type: PageOwnerType
   context_page_owner_id?: string
   subject?: string
 }
 
 /**
- * A Partner clicks on frame measurements fields in the CMS artworks edit page.
+ * A Partner clicks on framed measurements fields in the CMS artworks edit page.
  *
- * This schema describes events sent to Segment from [[clickedOnFrameMeasurements]]
+ * This schema describes events sent to Segment from [[clickedOnFramedMeasurements]]
  *
  *  @example
  *  ```
  *  {
- *    action: "clickedOnFrameMeasurements",
+ *    action: "clickedOnFramedMeasurements",
```

Comment:
@brendamanganelli this is the only change we made that we wanted to call out to you for confirmation -- we changed all terminology from "frame measurements" to "framed measurements" (note the `d` in framed) to align with how the UI/database have been referring to them. Let us know if this change is problematic for any of your work.